tests/test.ts

```diff
@@ -1,147 +1,187 @@
-import util from 'util';
-import test from 'ava';
-import {jsdom} from 'jsdom';
-import m from '.';
+import * as util from 'util';
+import test, {TestContext, Context} from 'ava';
+import * as jsdom from 'jsdom';
+import m from '../src'; // tslint:disable-line:import-name
 
 const isNode8orHigher = Number(process.versions.node.split('.')[0]) >= 8;
 
-const PromiseSubclassFixture = class extends Promise {};
-const ErrorSubclassFixture = class extends Error {};
+/* Currently out of order, see https://github.com/Microsoft/TypeScript/issues/15202
+class PromiseSubclassFixture<T> extends Promise<T> {}*/
+class ErrorSubclassFixture extends Error {}
+
+const document = new jsdom.JSDOM().window.document;
+const createDomElement = (el: string) => document.createElement(el);
+
+interface Test {is(value: any): boolean; fixtures: any[]; }
+const types = new Map<string, Test>([
+	['undefined',
+		{is: m.undefined, fixtures: [undefined]}
+	], ['null',
+		{is: m.null_, fixtures: [null]}
+	], ['string',
+		{is: m.string, fixtures: [
+			'🦄',
+			'hello world',
+			''
+		]}
+	], ['number',
+		{is: m.number, fixtures: [
+			6,
+			1.4,
+			0,
+			-0,
+			Infinity,
+			-Infinity
+		]}
+	], ['boolean',
+		{is: m.boolean, fixtures: [true, false]}
+	], ['symbol',
+		{is: m.symbol, fixtures: [Symbol('🦄')]}
+	], ['array',
+		{is: m.array, fixtures: [[1, 2], new Array(2)]} // tslint:disable-line:prefer-array-literal
```

Comment:
There are multiple fixtures and this is a single line.
I'm fine with single-lining if there's only one fixture, although I'd rather consistency overall,
Meaning that each type would look like:
```typescript
['typename',
    { is: m.typename, fixtures: [
        // there can be one or more fixtures here, but this would be consistent, as there are currently
        // tests with one fixture which are one lined, but also some that're multiple fixtures but also one lined
    ]}
]
```

Reply:
Agreed, makes perfect sense